src/java/test/org/apache/zookeeper/test/ReadOnlyModeTest.java

```diff
@@ -239,13 +243,13 @@ public void testSessionEstablishment() throws Exception {
     public void testSeekForRwServer() throws Exception {
 
         // setup the logger to capture all logs
-        Layout layout = Logger.getRootLogger().getAppender("CONSOLE")
+        Layout layout = org.apache.log4j.Logger.getRootLogger().getAppender("CONSOLE")
```

Comment:
why is this necessary?

Reply:
I few lines above there're org.apache.log4j references which conflicts with the 'Logger' class, hence it needs to be explicitly referenced.

Reply:
not going to push too hard on this, but I think you can just use log4j everywhere like in `QuorumPeerMainTest`

Reply:
slf4j's string formatter feature is being used in these tests. I wouldn't refactor them to be honest.

Reply:
I guess my real question is, do we need any of the changes in this file?

Reply:
Makes sense. I revert the changes.